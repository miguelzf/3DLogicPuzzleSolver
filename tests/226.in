X X - - - X
X - B - - -
- - - - R -
- - - - - -
- G - C - X
X - - - X X

X - - - X X
- - - - - X
- R - Y - -
- - - - - -
X - - - B -
X X - - - -

X X - - - X
X - - C - -
- - - - - -
- G - Y - -
- - - - - X
X - - - X X

