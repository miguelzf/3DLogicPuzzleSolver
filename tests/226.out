
Solution:
X  X  -  -  -  X  
X  -  B  b  b  b  
-  -  r  r  R  -  
-  -  r  -  c  c  
-  G  r  C  c  X  
X  g  r  -  X  X  
================
X  g  r  -  X  X  ||  X  X  c  -  b  X  
g  g  r  -  -  X  ||  X  -  c  C  b  -  
g  R  r  Y  y  y  ||  y  y  y  y  b  -  
g  g  g  g  g  g  ||  g  G  -  Y  b  -  
X  -  -  -  B  b  ||  b  b  b  b  b  X  
X  X  -  -  -  -  ||  X  -  -  -  X  X  
